urdf/astra_stereo_s_u3.xacro

```diff
@@ -1,17 +1,27 @@
 <?xml version="1.0"?>
 
+
+<!-- datasheet: https://www.sodavision.com/wp-content/uploads/2023/12/2023.07-astra-stereo-s-u3-datasheet-_v2.0_en.pdf -->
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
-    <xacro:property name="M_PI" value="3.14159"/> 
+    <xacro:property name="M_PI" value="3.14159"/>
+
+    <xacro:property name="orbbec_astra_stereo_s_u3_mass" value="0.05"/> <!-- estimated, needs to be checked -->
+    <xacro:property name="orbbec_astra_stereo_s_u3_length" value="0.0653"/>
+    <xacro:property name="orbbec_astra_stereo_s_u3_width" value="0.0123"/>
+    <xacro:property name="orbbec_astra_stereo_s_u3_height" value="0.0225"/>
+
+    <xacro:include filename="$(find hector_xacro_tools)/urdf/inertia_tensors.urdf.xacro" />
 
-    <xacro:macro name="orbbec_astra_stereo_s_u3" params="name:=default *origin *parent">
-        <joint name="astra_stereo_s_${name}_base_joint" type="fixed">
-            <child link="astra_stereo_s_${name}_base_link"/>
+    <xacro:macro name="${name}" params="name:=default *origin *parent">
```

Comment:
The macro name can not be a variable.

Reply:
The name is how you instantiate the macro, the name used later is here declared as a param of the macro.
We should probably also remove the default value "default" from name, since now it makes no sense to use the macro without specifying a name.